src/registry/views/static/component-info.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
```

Comment:
Yeah, I realised the UI wasn't covered by any tests so I managed to discover this manually. To facilitate testing, I created a separate PR to add a couple of test: #455 - so this should not happen again in the future ;)